### PR TITLE
Add caveat for parsing files from disk

### PR DESCRIPTION
**Pull request by whatnick**
The file parsing does not note encoding caveats. Included this for people using files with unicode encoding. A kludge for now till proper from_file method is introduced.

### CODE REVIEW

File: docs/usage_guide.rst [hunk 1]
```diff
@@ -102,7 +102,14 @@ You can create a KML object by reading a KML file as a string
 
     # Start by importing the kml module
     >>> from fastkml import kml
-
+    
+    #Read file into string and convert to UTF-8
+    >>> with open(kml_file, 'r') as myfile:
```
Comment (cleder) [inline docs/usage_guide.rst]: why not simply:
`with open(kml_file, "rt", encoding="utf-8") as myfile:`

File: docs/usage_guide.rst [hunk 2]
```diff
@@ -102,7 +102,14 @@ You can create a KML object by reading a KML file as a string
...
+    ...     data=myfile.read().replace('\n', '')
```
Comment (cleder) [inline docs/usage_guide.rst]: why?

Comment (whatnick) [reply docs/usage_guide.rst]: Sure newline replacements are unnecessary. It was just some other code for kml reading I had lying around.

**Comment by whatnick** [comment]
Changed patch branch to open and encode

**Pull request merged**